EXILED/Exiled.CustomItems/API/Features/CustomItem.cs

```diff
@@ -666,10 +667,6 @@ public virtual uint Spawn(IEnumerable<SpawnPoint> spawnPoints, uint limit)
                         break;
                     }
                 }
-                else if (spawnPoint is RoleSpawnPoint roleSpawnPoint)
-                {
-                    Spawn(roleSpawnPoint.Role.GetRandomSpawnLocation().Position, null);
-                }
```

Comment:
Why?

Reply:
RoleSpawnPoint.Position do the same.

Reply:
https://github.com/ExMod-Team/EXILED/blob/bbae123eeacf217bc818fa5de4c57b3c80a16a12/EXILED/Exiled.API/Features/Spawn/RoleSpawnPoint.cs#L41-L47